scripts/schema.sql

```diff
@@ -1,13 +1,13 @@
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 
 CREATE TABLE todos (
-    id uuid DEFAULT uuid_generate_v4 (),
+	id uuid DEFAULT uuid_generate_v4 (),
 	list_id uuid,
 	todo VARCHAR(255) NOT NULL,
-	done BOOLEAN,
+	done BOOLEAN DEFAULT false,
 	created_at TIMESTAMP DEFAULT current_timestamp,
 	updated_at TIMESTAMP DEFAULT current_timestamp,
-    PRIMARY KEY (id)
+	PRIMARY KEY (id)
```

Comment:
This whitespace made copying and pasting the schema into `psql` break. So i redid it and it works.   ??

Reply:
Oh and not having default false on done was causing issues.